blog/content/post/2020-03-24-trenchboot-nlnet-introduction.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Open Source DRTM with TrenchBoot for AMD processors. Introduction.
+abstract: This article starts an entire series of articles related to title
+          project. Today, you will find out why we have started such project and
```

Comment:
As in other post I suggest to not use time-related words. Please use something like: `by reading this blog post, you will find out ...`

Reply:
done